firmware/thruna/DIM/src/app/app_sevenSegDisplays.c

```diff
@@ -83,4 +83,28 @@ void app_sevenSegDisplays_disableGroup(SevenSegGroup group)
     io_sevenSegDisplays_disable(group, SEVEN_SEG_SUBPOSITION_L);
     io_sevenSegDisplays_disable(group, SEVEN_SEG_SUBPOSITION_M);
     io_sevenSegDisplays_disable(group, SEVEN_SEG_SUBPOSITION_R);
+}
+
+// the faultcoding we are getting from can is basically looking at fault codes which have a count of  [1,INF)
+// and cycle through the warinings or faults
+
+bool app_sevenSegDisplays_setFaultCode(SevenSegGroup group, uint16_t fault_code)
+{
+    if (fault_code > 600)
+    // If the number inserted is bigger than 600 which is the biggest fault/warning code possible
+    // as the maximum faultcode possible to enter is 552
+    {
+        return false;
+    }
```

Comment:
I would delete this condition - it might end up causing a headache down the line because 600 being the max might change.

Reply:
Ok sounds good